components/custom-button/custom-button.style.js

```diff
@@ -0,0 +1,67 @@
+import styled, { css } from 'styled-components';
+
+const InvertedButtonStles = css`
+  background-color: white;
+  border: none;
+  color: black;
+  &:hover {
+    background-color: black;
+    color: white;
+    @media screen and (max-width: 800px) {
+      transition: unset;
+      background-color: white;
+      color: black;
+      transform: unset;
+    }
+  }
+`;
+
+const primaryStyle = css`
+  background-color: #ee1a75;
+  border: none;
+  color: white;
+  &:hover {
+    /* background-color: white;
+    border: 1px solid #4285f4;
+    color: #4285f4;
+    transform: translate(0, -10%);
+    transition: all 0.2s; */
+  }
```

Comment:
We can remove this if not needed.

Reply:
i will implement this , just thinking about the color scheme